docs/getting-started/0-agent.md

```diff
@@ -13,37 +13,241 @@ Before initializing your agent, make sure you have followed the setup guide for
+
+#### Local network: 
+
+If you are going to test with a local network you have to download the genesis file corresponding to the networks local ledger.
+Example: [DTS Verifiable Credential Issuer Service](https://github.com/bcgov/dts-vc-issuer-service)
+Corresponding genesis file: http://test.bcovrin.vonx.io/genesis
+
+Code for downloading genesis file
+
+```ts
+//Function for downloading genesis file and return string value
+async function downloadGenesis(genesisUrl: string) {
+  try {
+    const response = await axios.get(genesisUrl); //Axios is a lightweight HTTP client, you can use any other library
+    return response.data;
+  } catch (e) {
+    console.log('EXCEPTION> downloadGenesis:', e);
+    return null;
+  }
+}
+
+
+//Function for storing genesis string value to a local file on the device 
+async function storeGenesis(
+  genesis: string,
+  fileName: string,
+): Promise<string> {
+  try {
+    const genesisPath = `${RNFS.DocumentDirectoryPath}/${fileName}`;
+    await RNFS.writeFile(genesisPath, genesis, 'utf8');
+    return genesisPath;
+  } catch (e) {
+    console.log('EXCEPTION> storeGenesis:', e);
+    return '';
+  }
+}
+
+//Somewhere in the code (before init Agent)
+const genesis = await downloadGenesis(GENESIS_URL);
+const genesisPath = await storeGenesis(genesisString, 'genesis.txn');
+
+```
+
+Note: You do not need the genesis file if you are creating a connection between your Agent and another Agent for exchanging simple messages.
+
+### 3- Get Mediator Connection URL (Optional)
+
+Mediators (Routing Agents) are Agents that serve as intermediaries to facilitate the flow of messages between other types of agents.
+You will need a mediator Agent if you are going to deal with VC (Verifiable Credentials), however, you can ignore the mediator step if you are creating an Agent for the sole purpose of exchanging messages with another Agent. 
+
+Example: If you are testing VC related functionality and need a mediator, you can use the [Indicio Public Mediator](https://indicio-tech.github.io/mediator/).
+- Head to [Indicio Public Mediator](https://indicio-tech.github.io/mediator/).
+- Copy mediator invite url and save it (i.e. MEDIATOR_INVITE = "url").
+
+Note: If the invite url uses insecure http you will have to adjust your code to allow for insecure traffic.
+- Instructions for [iOS](https://stackoverflow.com/questions/31254725/transport-security-has-blocked-a-cleartext-http).
+- Instructions for [Android](https://stackoverflow.com/questions/51902629/how-to-allow-all-network-connection-types-http-and-https-in-android-9-pie).
+
+More about [Mediators](3-routing.md).
+
+### 4- Create Agent
+
+```ts
+const agentConfig: InitConfig = {
+      label: 'my-agent9',
+      mediatorConnectionsInvite: MEDIATOR_INVITE,
+      walletConfig: {id: 'walletId10'},
+      walletCredentials: {key: 'testkey0230482304823042304244'},
+      autoAcceptConnections: true,
```

Comment:
We now have an auto accept "ContentApproved" that I think is good enough for the getting started guide. We could then explain how to manually handle the flow in the in-depth guides

```suggestion
      autoAcceptConnections: true,
      autoAcceptCredentials: AutoAcceptCredential.ContentApproved,
      autoAcceptProofs: AutoAcceptProof.ContentApproved,
```